### DISCUSSION TITLE
It's a good idea to add BrashMonkey's "Spriter Pro" (2D game animation tool) to the extensions

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#945] [Ideas]
see
https://brashmonkey.com/spriter-pro/

Advantage:
You can use a free version on your game and  Spriter Pro is cheaper as Spline.
Also performance should be good and a new version is planed too: Spriter 2

Here are some github sources for first start.
https://github.com/j2doll/test-ccspriterx  (MIT)
https://github.com/lucidspriter/SpriterPlusPlus  (zlib)

**Comment by aismann** [comment]
First result:

[Show me the clip (youtube)](https://youtu.be/oR4eCCZ6xCQ)

![image](https://user-images.githubusercontent.com/8652787/233795726-04491420-feee-4e18-a6b5-d3a7def739c8.png)

Another more complex example (only for testing/bug finding issues (maybe it has copyrights):
[Show me the clip (youtube)](https://youtu.be/meKycTd2j6I)

![image](https://user-images.githubusercontent.com/8652787/233831135-add116ab-d854-4440-a6df-804a2e5704c8.png)

**Comment by paulocoutinhox** [comment]
Nice! It was added? 

The format is SCML?

Ty.

**Reply by aismann** [reply]
> Nice! It was added?
> 
> The format is SCML?
> 
> Ty.

A long time ago...
I never added ‘Spriter Pro’ to Axmol.
Yes, SCML.
